sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala

```diff
@@ -245,7 +246,7 @@ class CacheManager extends Logging {
       // node, so that the returned caching plan won't replace the hint node and drop the hint info
       // from the original plan.
       case hint: ResolvedHint => hint
-
+      case command: IgnoreCachedData => command
```

Comment:
nit: move it to the first case so that it's more obvious we ignore it
```
plan transformDown {
  case p: IgnoreCachedData => p
  ...
```

Reply:
Agreed, reordered match-case